services/subscription.go

```diff
@@ -409,11 +427,23 @@ func (le InitiatorSubscriptionLogEvent) ContractPayment() (*assets.Link, error)
 	return payment, nil
 }
 
+// Requester pulls the requesting address out of the LogEvent's topics.
+func (le InitiatorSubscriptionLogEvent) Requester() common.Address {
+	b := le.Log.Topics[RunLogTopicRequester].Bytes()
+	return common.BytesToAddress(b)
+}
+
+func parseRequestID(data hexutil.Bytes) string {
+	parsedID := []byte(data)[:evmWordBytesSize]
```

Comment:
Rather than slicing here, could we accept a fixed size input like `common.Hash`?